code/modules/food&drinks/drinks/drinks/bottle.dm

```diff
@@ -33,7 +33,7 @@
 	qdel(src)
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/attack(mob/living/target as mob, mob/living/user as mob)
-
+	..()
```

Comment:
Not as far as I know. It calls the parent's attack proc, which contains the code to deal with heated items.